tools/ConvertTo-PseudoLocalization.ps1

```diff
@@ -117,11 +114,12 @@ function GetPseudoLocalization([string]$key, [string]$value, [string]$comment) {
         }
 
         # Add 40% padding to the end of the string
-        $paddingLength = [System.Math]::Round(0.4 * $_.Length)
+        $paddingLength = [System.Math]::Round(0.4 * $newValue.Length)

```

Comment:
`$_` is the string with the single-character placeholders. The number of `!!!` at the end however corresponds to the original string length, so we need to use `$newValue` here (its length should be identical to the original `$value` length.